src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/ccd/CcdAuthenticatorFactory.java

```diff
@@ -30,7 +27,7 @@ public CcdAuthenticatorFactory(
 
     public CcdAuthenticator createForJurisdiction(String jurisdiction) {
         Credential user = users.getUser(jurisdiction);
-        log.info("getting Idam Open Id Access token ");
```

Comment:
we can keep this log might be useful for future debugging ?

Reply:
we can but it is not saying too much. I have added to distinguish from previous version in deployments.